api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -421,6 +421,52 @@ type OIDCIssuerProfile struct {
 	Enabled *bool `json:"enabled,omitempty"`
 }
 
+// MarketplaceExtension represents the configuration for a marketplace extension.
+// See also [AKS doc].
```

Comment:
Nit: is this markdown-like anchor thing (`[AKS doc]`) a go comment idiom?

Reply:
Yes. Jon actually just explained this [here](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4316#discussion_r1456552607).

Reply:
TIL!